### ISSUE TITLE
Can't run argcomplete on zsh

### ISSUE THREAD

**Issue by eyalev** [kislyuk/argcomplete#135]
Hi,

I'm trying to run `argcomplete` with `zsh` on a fresh ubuntu:14.04 Docker container.

`# First trying using bash`
Enabled bash completion
Installed python, pip
pip install argcomplete
Created [describe_github_user.py](https://github.com/kislyuk/argcomplete/blob/master/docs/examples/describe_github_user.py) in `/root`
chmod +x describe_github_user.py
`activate-global-python-argcomplete`
Running `./describe_github_user.py --organization heroku --member <TAB>` works :+1:

`# Now trying zsh`
Installed `zsh`
Moved to `zsh` shell
In `.zshrc`:
````
autoload -Uz compinit bashcompinit
compinit
bashcompinit

source /etc/bash_completion.d/python-argcomplete.sh

eval "$(register-python-argcomplete /root/describe_github_user.py)"
````

Running `./describe_github_user.py --organization heroku --member <TAB>`

Not working :(

What I'm doing wrong?

Thanks

**Comment by kislyuk** [comment]
I'm not a zsh expert, but I just tried the exact commands you pasted in my zsh and it worked fine. My zsh version is `zsh 5.2 (x86_64-apple-darwin15.4.0)`.

**Comment by eyalev** [comment]
Ok, for some reason the `./` in the command was giving me problems.

I added the `root` directory to `PATH`:
`export PATH=/root:$PATH`

And then:
`describe_github_user.py --organization heroku --member <TAB>` worked. 

Yay :)

Maybe it's an Ubuntu specific thing..

If that's the case it might be worth mentioning in the README in some way. Could have saved me a lot of time.

----

Some backstory info: 

I knew it had to work somehow because I use Google's `gcloud` tool which uses `argcomplete`.

https://cloud.google.com/sdk/docs/quickstart-linux
https://github.com/eyalev/gcloud/blob/master/completion.zsh.inc

So I tried to find out what is different when I'm using `gcloud` and I got to the above "fix".
